bin/start.spec

```diff
@@ -20,10 +20,8 @@ a = Analysis(['../start.py'],
              "gevent", "geventwebsocket", "flask_socketio", "newspaper", "vaderSentiment", "sentry_sdk",
              "vaderSentiment.vaderSentiment",
              "aiofiles",
-             "ccxt", "ccxt.async_support", "cryptofeed", "cmath", "cryptography", "websockets", "yarl", "idna", "sortedcontainers",
-             "websockets.legacy", "websockets.legacy.auth", "websockets.legacy.client", "websockets.legacy.compatibility",
-             "websockets.legacy.framing", "websockets.legacy.handshake", "websockets.legacy.http", "websockets.legacy.protocol",
-             "websockets.legacy.server"
```

Comment:
no need for it anymore ? 😮 🎉

Reply:
Yes, websockets legacy are not needed anymore, but we need a new one : order_book.